frontend/src/components/NavBar.js

```diff
@@ -1,17 +1,92 @@
-/**
-    Super basic navbar without any styles
- */
-import React from "react";
+import React, { useState } from "react";
+import { NavLink } from "react-router-dom";
+import Modal from "react-modal";
 import { SITE_PAGES } from "../constants/links";
+import Search from "./Search";
+import "../styles/NavBar.css";
+
+// Pages to display in the NavBar. If adding more pages, adjust Page Links media query in NavBar.css
+const PAGES = {
+  "Wish Granting": SITE_PAGES.WISH_GRANTING,
+  Manage: SITE_PAGES.MANAGE,
+};
+
+Modal.setAppElement(document.getElementById("#root"));
+
+/*
+    NavBar component, which is at the top of each page and provides links to navigate between each page. 
+    Also contains the file search bar and the account menu to go to the profile page or sign out.
+*/
+function NavBar() {
+  const [dropdown, setDropdown] = useState(false);
 
-export default function NavBar() {
   return (
-    <div>
-      {/* Navigation Links */}
-      <a href={SITE_PAGES.LOGIN}>Log In</a>
-      <a href={SITE_PAGES.PEOPLE}>People</a>
-      <a href={SITE_PAGES.WISH_GRANTING}>Wish Granting</a>
-      <a href={SITE_PAGES.PROFILE}>Profile</a>
-    </div>
+    <nav className="container">
+      <ul className="navlist">
+        {/* Container for Logo and page links */}
+        <li className="logo-and-pages">
+          <li className="logo-container">
+            <NavLink to={SITE_PAGES.MANAGE}>
+              <img src="/img/maw_logo.png" alt="Make-A-Wish logo" className="maw-logo" />
+            </NavLink>
+          </li>
+
+          <li className="pages-container">
+            {Object.entries(PAGES).map(([page, route]) => (
+              <NavLink className="page-links" activeClassName="underline" to={route}>
```

Comment:
React also gives an error if you don't give every element inside `map` a unique key prop (also helps with performance)

```suggestion
              <NavLink key={route} className="page-links" activeClassName="underline" to={route}>
```